test/src/built_in/layout/standard/toast/minimal/minimal_standard_toast_test.dart

```diff
@@ -0,0 +1,165 @@
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            icon: Icon(Icons.message),
+            onCloseTap: () {},
+          ),
+        ),
+      );
+
+      expect(find.byIcon(Icons.message), findsOne);
+    });
+
+    testWidgets(
+        'should not render LinearProgressIndicator if theme has showProgressBar false',
+        (tester) async {
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            progressBarValue: 0.5,
+            onCloseTap: () {},
+          ),
+          showProgressBar: false,
+        ),
+      );
+
+      expect(find.byType(LinearProgressIndicator), findsNothing);
+    });
+
+    testWidgets(
+        'should render with LinearProgressIndicator if progressBarValue is provided',
+        (tester) async {
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            progressBarValue: 0.5,
+            onCloseTap: () {},
+          ),
+          showProgressBar: true,
+        ),
+      );
+
+      expect(find.byType(LinearProgressIndicator), findsOne);
+    });
+
+    testWidgets('should render with custom progress bar widget',
+        (tester) async {
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            progressBarWidget: CircularProgressIndicator(),
+            onCloseTap: () {},
+          ),
+          showProgressBar: true,
+        ),
+      );
+
+      expect(find.byType(CircularProgressIndicator), findsOne);
+    });
+
+    testWidgets('should not render blur effect if not enabled in theme',
+        (tester) async {
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            progressBarWidget: CircularProgressIndicator(),
+            onCloseTap: () {},
+          ),
+          shouldApplyBlurEffect: false,
+        ),
+      );
+
+      expect(find.byType(BackdropFilter), findsNothing);
+    });
+
+    testWidgets('should render blur effect if enabled in theme',
+        (tester) async {
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            progressBarWidget: CircularProgressIndicator(),
+            onCloseTap: () {},
+          ),
+          shouldApplyBlurEffect: true,
+        ),
+      );
+
+      expect(find.byType(BackdropFilter), findsOne);
+    });
+
+    testWidgets('should trigger onCloseTap callback', (tester) async {
+      bool isCloseTapped = false;
+
+      await tester.pumpWidget(
+        buildWithToastificationTheme(
+          MinimalStandardToastWidget(
+            onCloseTap: () {
+              isCloseTapped = true;
+            },
+          ),
+        ),
+      );
+
+      await tester.tap(find.byKey(ValueKey('close_button_visible')));
+
+      expect(isCloseTapped, true);
+    });
+  });
+}
```

Comment:
I’d also recommend covering other scenarios related to the theme and Toastification styles.
If you’re interested, you could add these test cases as well!